wiki/Tournaments/CFV/1st/en.md

```diff
@@ -0,0 +1,489 @@
+
+Friday, 8 Dec 2023:
+
+| Team A |  |  | Team B | Match link | VOD link |
+| --: | :-: | :-: | :-- | :-- | :-- |
+| oid45 | 1 | **4** | **RealF233** | [#1](https://osu.ppy.sh/community/matches/111624822) |  |
+| **Saemitsu** | **4** | 0 | Kuk2Ru | [#1](https://osu.ppy.sh/community/matches/111624765) | [#1](https://www.bilibili.com/video/BV17G411e7tz) |
+
+Saturday, 9 Dec 2023:
+
+| Team A |  |  | Team B | Match link | VOD link |
+| --: | :-: | :-: | :-- | :-- | :-- |
+| Endless fare | 2 | **4** | **ILikeTea** | [#1](https://osu.ppy.sh/community/matches/111639911) | [#1](https://www.bilibili.com/video/BV11j411W7Rp) |
+| **KimMui** | **4** | 0 | oitre gouqi233 | [#1](https://osu.ppy.sh/community/matches/111641135) | [#1](https://www.bilibili.com/video/BV1wc411q79x/) |
+| Mon3tr_Miku | 1 | **4** | **noglawitch** | [#1](https://osu.ppy.sh/community/matches/111641122) | [#1](https://www.bilibili.com/video/BV1FQ4y1G7QG/) |
+| **Axium0w0** | **4** | 0 | qiufeng_xz | [#1](https://osu.ppy.sh/community/matches/111642859) | [#1](https://www.bilibili.com/video/BV15w41187kd/) |
+
+Saturday, 10 Dec 2023:
+
+| Team A |  |  | Team B | Match link | VOD link |
+| --: | :-: | :-: | :-- | :-- | :-- |
+| **- Sky -** | **4** | 1 | Crogzy Wogzy | [#1](https://osu.ppy.sh/community/matches/111653137) | [#1](https://www.bilibili.com/video/BV1zu4y1G7p5/) |
+| **frankgh3** | **0** | -1 | Ballistic | *win by default* |  |
+| Jugemu-Jugemu | -1 | **0** | **GOSChara** | *win by default* |  |
+| mister_Andres | 0 | **4** | **aabc271** | [#1](https://osu.ppy.sh/community/matches/111657519) | [#1](https://www.bilibili.com/video/BV1wc411q7eo) |
+| **AbstractApple** | **4** | 0 | Catastrophe | [#1](https://osu.ppy.sh/community/matches/111657886) | [#1](https://www.bilibili.com/video/BV11w41147LZ) |
+| **PolarSwader** | **4** | 0 | RhythmBoii | [#1](https://osu.ppy.sh/community/matches/111658342) | [#1](https://www.bilibili.com/video/BV1di4y1v77j) |
+| **kissnolook** | **4** | 0 | redhe | [#1](https://osu.ppy.sh/community/matches/111660117) | [#1](https://www.bilibili.com/video/BV1ZQ4y137G3) |
+| [AR]bfdgbzfhs | 1 | **4** | **Chrisse** | [#1](https://osu.ppy.sh/community/matches/111660160) |  |
+| **[GB]ChickenGold** | **4** | 0 | huypham_gm | [#1](https://osu.ppy.sh/community/matches/111660768) | [#1](https://www.bilibili.com/video/BV1qH4y1k7L6) |
+| **Fuitje** | **4** | 1 | SpikyPT | [#1](https://osu.ppy.sh/community/matches/111662780) | [#1](https://www.bilibili.com/video/BV1cj411s73Y) |
+
+
+## Ruleset
+
+The ruleset is primarily based on rulesets used in the [osu!mania 4K World Cup 2024](/wiki/Tournaments/MWC/2024_4K#ruleset) and [4 Digit osu!mania World Cup 2024](/wiki/Tournaments/4DM/2024#ruleset).
+
+- Any general rules not shown below but listed in the reference tournament are still valid.
+
+### Tournament rules
```

Comment:
since this section is long i've got the entire rules reviewed here (you can just copy and paste) :

--> http://pastebin.com/NV4NwLEs